src/labeler.ts

```diff
@@ -40,6 +40,23 @@ export async function run() {
 
     const labels: string[] = [];
     const labelsToRemove: string[] = [];
+
+    if (!changedFiles.length) {
+      core.debug(`exiting early because pr #${prNumber} has no changed files.`);
```

Comment:
This should be a warning. Creating an empty pr should definitely be noticed by the user.